pkg/mac/BoE-Info.plist

```diff
@@ -239,5 +239,7 @@
 			</dict>
 		</dict>
 	</array>
+	<key>NSHighResolutionCapable</key>
```

Comment:
I understand that ultimately this is a good thing to do at some point, but what effect does it have? On my computer I can't see any difference. I assume it does make a difference on your computer though.

Reply:
When I added this flag, I noticed some small changes, https://developer.apple.com/library/archive/documentation/GraphicsAnimation/Conceptual/HighResolutionOSX/Explained/Explained.html . But it is not impossible that the new versions of the system have made these changes minimal.

Reply:
Even after setting this flag, it appears the game is locked to "low resolution":

<img width="176" alt="image" src="https://user-images.githubusercontent.com/578731/213931263-06a01944-af8a-43dc-a261-5ab241238668.png">

(That's from selecting the .app in the Finder and choosing Get Info.)

Reply:
Unsure, on OSX 12.6.1, I obtain for instance:
<img width="269" alt="Capture d’écran 2023-01-23 à 09 52 09" src="https://user-images.githubusercontent.com/13163168/213998964-85ddeaf8-901c-4657-a265-5bca40961213.png">
which was created with
<img width="792" alt="Capture d’écran 2023-01-23 à 10 04 12" src="https://user-images.githubusercontent.com/13163168/214001372-40a15564-438a-45a3-98f7-fdbb845dba02.png">